nixos/modules/system/boot/loader/limine/limine-install.py

```diff
@@ -203,7 +204,10 @@ def copy_file(from_path: str, to_path: str):
     if not os.path.exists(dirname):
         os.makedirs(dirname)
 
-    shutil.copyfile(from_path, to_path)
+    shutil.copyfile(from_path, to_path + ".tmp")
```

Comment:
I like the idea! But can't this lead to situations where
- the script stops after this line
- someone updates nixpkgs
- the new file has a new name (new_kernel.tmp)
- the old tmp file is never removed?

Reply:
1. That is the issue resolved by this change
2. I don't understand what you mean by that
3. That won't happen (hopefully)
4. It's overwritten